test/JDBC/src/main/resources/config.txt

```diff
@@ -49,4 +49,4 @@ testFileRoot = ./
 isUpgradeTestMode = false
 
 # Default SLA in milliseconds
-defaultSLA = 40000
+defaultSLA = 60000
```

Comment:
Do we have an agreement(with dev-team) to increase it?

Reply:
No, this was a temp change. 
Increased the SLA for a particular test file.